models/comments.model.js

```diff
@@ -0,0 +1,25 @@
+const db = require("../db/connection")
+
+function selectCommentsByArticleId(article_id, sort_by = "created_at", order = "DESC") {
+    const validSortBy = ["created_at", "votes", "author"]
+    const validOrder = ["ASC", "DESC"]
+
+    if(!validSortBy.includes(sort_by)){
+        return Promise.reject({status: 400, msg: "bad request"})
+    }
+
+    if(!validOrder.includes(order)){
+        return Promise.reject({status: 400, msg: "bad request"})
+    }
+
```

Comment:
It's great you have added this sorting functionality but it isn't needed on this ticket. If you do want to keep it, it's ok though. However, please do add the missing test I mentioned